application/database/migrations/2023_04_03_122625_add_file_type_classifier_values.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use App\Enums\ClassifierValueType;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        DB::table('classifier_values')->insert(
+            array_map(function (array $classifierValueData) {
+                return [
+                    ...$classifierValueData,
+                    'type' => ClassifierValueType::FileType->value,
+                    'meta' => '[]',
```

Comment:
**[non-blocker]** Hmm, I notice that while the `meta` column is nullable, you are still setting it to an empty array. Not a big deal, but perhaps if the plan is to always default to an empty array, then the column shouldn’t be nullable?